client/server/render/index.js

```diff
@@ -219,9 +219,8 @@ export function serverRender( req, res ) {
 	attachI18n( context );
 
 	if ( shouldServerSideRender( context ) ) {
-		cacheKey = `${ getNormalizedPath( context.pathname, context.query ) }:gdpr=${
-			context.showGdprBanner
-		}`;
+		cacheKey = getCacheKey( req );
```

Comment:
Any chance `context.pathname` and `context.query` would differ from `req.pathname` and `req.query`?

Reply:
It looks like context is set like so:

https://github.com/Automattic/wp-calypso/blob/5d4d01ccde94a0a4b98f60800a22f6db906006f2/client/server/isomorphic-routing/index.js#L71-L73

So it should be the same. 

We also already used request.path and request.query when we _get_ the cache, so I'm unsure why it was ever different